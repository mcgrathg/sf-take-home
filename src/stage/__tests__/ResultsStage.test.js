import React from 'react';
import { render } from 'react-testing-library';
import ResultsStage from '../ResultsStage';

describe('ResultsStage', () => {
  it('renders', () => {
    const { container } = render(
      <ResultsStage showOff={[]} downplay={[]} comfortable={[]} />,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
