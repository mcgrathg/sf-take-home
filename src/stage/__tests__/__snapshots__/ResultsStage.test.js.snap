// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsStage renders 1`] = `
.c1 {
  margin: 0 auto 1rem;
  color: #2f3237;
  font-size: 1.5rem;
  line-height: 1.75;
  font-weight: normal;
}

.c2 {
  position: relative;
  font-weight: 800;
}

.c2::after {
  position: absolute;
  left: 0;
  bottom: 0;
  height: 0.5rem;
  width: 100%;
  content: '';
  background-color: #ccf2f0;
  z-index: -1;
  -webkit-transition: all 0.5s;
  transition: all 0.5s;
  -webkit-transform-origin: left top;
  -ms-transform-origin: left top;
  transform-origin: left top;
  -webkit-transform: scale(0,1);
  -ms-transform: scale(0,1);
  transform: scale(0,1);
}

.c2.mounted::after {
  -webkit-transform: scale(1,1);
  -ms-transform: scale(1,1);
  transform: scale(1,1);
}

.c0 {
  padding-bottom: 2rem;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

<div
  style="opacity: 0;"
>
  <div
    class="c0"
    data-testid="show-off-results"
  >
    <h1
      class="c1"
    >
      You're 
      <span
        class="c2"
      >
        Showing Off
      </span>
      :
    </h1>
    <div
      class="c3"
    />
  </div>
  <div
    class="c0"
    data-testid="downplay-results"
  >
    <h1
      class="c1"
    >
      We'll 
      <span
        class="c2"
      >
        Downplay
      </span>
      :
    </h1>
    <div
      class="c3"
    />
  </div>
  <div
    class="c0"
    data-testid="other-results"
  >
    <h1
      class="c1"
    >
      You're comfortable about
      <span
        class="c2"
      />
      :
    </h1>
    <div
      class="c3"
    />
  </div>
</div>
`;
